projects/nutanix-cloud-native/cluster-api-provider-nutanix/Makefile

```
@@ -0,0 +1,30 @@
+BASE_DIRECTORY:=$(shell git rev-parse --show-toplevel)
+GIT_TAG:=$(shell cat GIT_TAG)
+GOLANG_VERSION:=$(shell cat GOLANG_VERSION)
```

can we initialize the `GOLANG_VERSION` here directly instead of reading from the file for consistency https://github.com/aws/eks-anywhere-build-tooling/blob/main/projects/kubernetes-sigs/cluster-api-provider-vsphere/Makefile#L3
```suggestion
GOLANG_VERSION?="1.17"
```

---

Sure, we can. With that said, I did what the docs said :) https://github.com/aws/eks-anywhere-build-tooling/blob/main/docs/development/add-new-project.md

---

fixed

---

thanks for pointing that out! I will update that doc